installer/src/main/bin/compareModel.cmd

```diff
@@ -0,0 +1,73 @@
+@ECHO OFF
+@rem **************************************************************************
+@rem modelHelp.cmd
+@rem
+@rem Copyright (c) 2020, Oracle Corporation and/or its affiliates.  All rights reserved.
+@rem Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+@rem
+@rem     NAME
+@rem       compareModel.sh - WLS Deploy tool to compare two models
+@rem
+@rem     DESCRIPTION
+@rem       This script compares two models. The models compared must be both yaml or both json
+@rem
+@rem This script uses the following variables:
+@rem
+@rem JAVA_HOME             - The location of the JDK to use.  The caller must set
+@rem                         this variable to a valid Java 7 (or later) JDK.
+@rem
+@rem
```

Comment:
The variables WLSDEPLOY_HOME and WLSDEPLOY_PROPERTIES are still relevant in jython-only scripts, just copy descriptions from modelHelp.sh and .py

Reply:
added